docs/testing/create-a-network-shared-folder-catalog-for-task-pane-and-content-add-ins.md

```diff
@@ -10,7 +10,7 @@ localization_priority: Priority
 You can install an Office Add-in for testing in an Office client running on Windows by publishing the manifest to a network file share (instructions below).
 
 > [!NOTE]
-> If your add-in project was created with the [Yeoman generator for Office Add-ins](https://github.com/OfficeDev/generator-office), there is an alternate way of sideloading the add-in that might work for you. For details, see [Sideload Office Add-ins using the sideload command](sideload-office-addin-using-sideload-command.md).
+> If your add-in project was created with a sufficiently recent version of the [Yeoman generator for Office Add-ins](https://github.com/OfficeDev/generator-office), the add-in will automatically sideload in the Office desktop client when you run `npm start`.
```

Comment:
Yes; I've asked the devs for this info (i.e., version number when this was introduced), but it may take them a bit to get back to me. I'll go ahead and merge this PR now (to get the doc updates out) and will plan to update this note with version number whenever I receive that info. Thanks for the review @ElizabethSamuel-MSFT!

Reply:
Actually, I'm going to hold off on merging this PR for a bit longer -- because one of the commands described (`npx office-addin-manifest validate MANIFEST_FILE`) doesn't work for me yet (seems like we may still be waiting on full deployment of that new 'validate' command).